### PR TITLE
Merging recent changes to Adonis' branch into dev

### PR DESCRIPTION
**Pull request by carolynrehm**

### CODE REVIEW

**Review by carolynrehm — COMMENTED** [review]
Everything looks good here, can we just rename the constraints in the sql setup file for the foreign key relationships so that they are a little more meaningful?

**Comment by carolynrehm** [comment]
Also @revaturelabs/batch-702-carolyn-rehm  I can't be the one to approve this, because I was the one to open it. It can be merged once we get an approval

**Review by AdonisCabreja — APPROVED** [review]

**Pull request merged**
